src/seml/commands/start.py

```diff
@@ -1189,9 +1216,9 @@ def prepare_experiment(
         cmd = get_shell_command(interpreter, os.path.join(temp_dir, exe), config)
 
     if not unobserved:
-        collection.update_one({'_id': exp_id}, {'$set': updates})
+        collection.update_one({'_id': exp['_id']}, {'$set': updates})
 
     # Print the command to be ran.
-    print(cmd)
+    print(f'{cmd} > {output_file} 2>&1')
     # We exit with 0 to signal that the preparation was successful.
     exit(0)
```

Comment:
I can't say I thoroughly checked that, I just trust you :)

Reply:
this just pipes stdout and stderr into a new file

Reply:
Was meant as a general comment for the new logic in `start.py`.